### PR TITLE
define the elements of list as str

### PR DESCRIPTION
**Pull request by tropicrainforest**
it's quick fix to make the Range in parameters work

### CODE REVIEW

**Comment by tropicrainforest** [comment]
i don't know what was wrong with ESGF, so I just made a bit of changes in its test based on the valid values to work.
The funny part of story is here:
https://github.com/FREVA-CLINT/freva/blob/main/src/evaluation_system/tests/parameters_test.py#L431
as you see Range issue has been fixed in unit test, but in the main code no!

File: src/evaluation_system/tests/esgf_test.py [hunk 1]
```diff
@@ -189,8 +190,8 @@ def test_freva_esgf_method(dummy_config):
 
 def test_find_files(capsys, search_dict, dummy_config):
     result_to_be = [
-        "output1/MPI-M/MPI-ESM-LR/decadal2000/mon/atmos/Amon/r1i1p1/tas/1/"
-        "tas_Amon_MPI-ESM-LR_decadal2000_r1i1p1_200101-201012.nc",
+        # "output1/MPI-M/MPI-ESM-LR/decadal2000/mon/atmos/Amon/r1i1p1/tas/1/"
+        # "tas_Amon_MPI-ESM-LR_decadal2000_r1i1p1_200101-201012.nc",
```
Comment (antarcticrainforest) [inline src/evaluation_system/tests/esgf_test.py]: Are these gone from ESGF? Can you double check with the maintainers?

Comment (tropicrainforest) [reply src/evaluation_system/tests/esgf_test.py]: hmm, who i need to check with him? Let me rerun the ci on working days, it might be fixed ;)

Comment (tropicrainforest) [reply src/evaluation_system/tests/esgf_test.py]: As I suspected, the ESGF test-related data is only available on weekdays. Beautiful!

**Comment by eelucio** [comment]
lgtm

**Review by antarcticrainforest — APPROVED** [review]
LGTM!

**Pull request merged**